src/Core/src/Core/IShadow.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Microsoft.Maui.Graphics;
+
+namespace Microsoft.Maui
+{
+	public interface IShadow
+	{
+		double Radius { get; }
+
+		double Opacity { get; }
+
+		Paint Paint { get; }
```

Comment:
Should we include some comments around this?. Paint could be a gradient, etc and for now we only support a solid color.

Reply:
Done!